client/package.json

```diff
@@ -9,6 +9,7 @@
     "@testing-library/user-event": "^7.1.2",
     "react": "^16.13.0",
     "react-dom": "^16.13.0",
+    "react-router-dom": "^5.1.2",
```

Comment:
Are you able to fix the yarn warnings easily? I recognize that these are not from this PR though.

```
warning " > @testing-library/user-event@7.2.1" has unmet peer dependency "@testing-library/dom@>=5".
warning "react-scripts > eslint-config-react-app@5.2.0" has incorrect peer dependency "eslint-plugin-flowtype@3.x".
warning "react-scripts > @typescript-eslint/eslint-plugin > tsutils@3.17.1" has unmet peer dependency "typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta".
```

Reply:
The reason I care is that it hides test output when running `mvn test` from the intellij plugin. Maybe there is an alternate fix, like hiding yarn errors from mvn...

Reply:
Those yarn install warning really only show up on the first install. Subsequent `mvn tests` won't show those warnings.